CampaignConfig/mdc2020_beam.cfg

```diff
@@ -698,6 +723,77 @@ job_setup.getconfig = True
 job_setup.multifile = False
 job_setup.setup_local = True
 
+[stage_muminusstops_cat_fcl]
```

Comment:
I thought we agreed not to make this explicit, since the real concatenation is done in the split and this is just copying 1 file -> 1 file.  Or maybe I'm missing something?

Reply:
I thought we agreed we were going to not do the concatenation in the split anymore, but maybe I misunderstood. If that's the case i'll remove the stage.

Reply:
Ok, I removed that.